### PR TITLE
Added CHANGELOG for Quickwit 0.6

### PR DESCRIPTION
**Pull request by fulmicoton**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
+- New columnar format:
+    - Fast fields can now have any cardinality (Optional, Multivalued, restricted). In fact cardinality is now only used to format the output.
+    - Dynamic Fields are now fast fields.
+- Elasticsearch/Opensearch compatible API.
+- String fast fields now can be normalized.
+- Various parameters of object storages can now be configured.
+- The ingest API makes it possible to force a commit
+- Ability to parse non-JSON data using VRL to extract some structure from documents.
+- Object storage can now use the .
```
Comment (fmassot) [inline CHANGELOG.md:33]: "." is a cryptic feature. Do you mean "Virtual-hosted–style requests"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- <Aggregation additions>
+- Added support for Prefix Phrase query.
+- The query language now supports different date formats.
```
Comment (fmassot) [inline CHANGELOG.md:36]:
```suggestion
- The query language now supports different date formats.
- Support hexadecimal input/ouput format for the bytes field
```

File: CHANGELOG.md (hunk 1 above)
Comment (fulmicoton) [reply CHANGELOG.md:36]: >.< yes that's what I meant.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +20,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [0.6.0] - 2023-06-03
+
+### Added
+- Elasticsearch/Opensearch compatible API.
+- New columnar format:
+    - Fast fields can now have any cardinality (Optional, Multivalued, restricted). In fact cardinality is now only used to format the output.
+    - Dynamic Fields are now fast fields.
+- String fast fields now can be normalized.
+- Various parameters of object storages can now be configured.
+- The ingest API makes it possible to force a commit
```
Comment (imotov) [inline CHANGELOG.md:34]: Maybe add something like "or wait for scheduled commit to occur".

**Pull request merged**